i6_models/parts/conformer/feedforward.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+import torch
+from torch import nn
+
+
+class ConformerPositionwiseFeedForward(nn.Module):
+    """
+    Conformer feedforward module
+    """
+
+    def __init__(self, input_dim: int, hidden_dim: int, dropout: float = 0.1):
+        """
+        :param input_dim: input dimension
+        :param hidden_dim: hidden dimension (normally set to 4*input_dim as suggested by the paper)
+        :param dropout: dropout probability
+        """
+        super().__init__()
+
+        self.linear_ff = nn.Linear(in_features=input_dim, out_features=hidden_dim, bias=True)
+
+        self.swish_activation = nn.SiLU()
+
+        self.linear_out = nn.Linear(in_features=hidden_dim, out_features=input_dim, bias=True)
+
+        self.dropout = nn.Dropout(p=dropout)
+
+    def forward(self, tensor: torch.Tensor):
+        """
+        :param torch.Tensor tensor: input tensor of shape [B,T,F]
+        :return: torch.Tensor of shape [B,T,F]
+        """
```

Comment:
Typing still misses the return value.
Typing in the docstring is also not necessary and should be avoided.
I would also add that `F` refers to the `input_dim`.
```suggestion
    def forward(self, tensor: torch.Tensor) -> torch.Tensor:
        """
        :param tensor: shape [B,T,F], F=input_dim
        :return: shape [B,T,F], F=input_dim
        """
```